README.md

```diff
@@ -1,2 +1,51 @@
-Hello-World
-In this project, you will set up a "Hello world" repository.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> This project is my first ever on microverse and its a linter setup as well as a hello world
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] README file is very important to provide along with your application or any other code you upload to GitHub. It actually gives someone potentially interested in your repository a brief overview of what your project is about, how to use it, what is its architecture, and so on. I think [providing your UI screenshots in README](https://medium.com/@justynagolawska/how-to-easily-add-screenshots-into-your-readme-file-on-github-d806a01d6ffd) is very useful.